docs/app-portal.mdx

```diff
@@ -76,19 +78,23 @@ curl -X POST "https://api.svix.com/api/v1/auth/dashboard_access/app_Xzx8bQeOB1D1
 </TabItem>
 </CodeTabs>
 
-
 ## Embedding in your own dashboard
 
 The returned URL from the previous section can also be embedded in your own dashboard using an iframe.
 
-<img alt="Embedded Management UI screenshot" src={useBaseUrl("img/embedded-management-ui.png")} />
+<img
+  alt="Embedded App Portal screenshot"
+  src={useBaseUrl("img/embedded-management-ui.png")}
+/>
 
 To add this to your application, just pass the URL you received in the previous example to the `src` property of the iframe:
 
 ```html
 <iframe
-    src="http://app.svix.com/login#key=eyJhcHBJZCI6ICJhcHBfMXRSdFlMN3pwWWR2NHFuWTRRZFI1azE4eXQ0IiwgInRva2VuIjogImFwcHNrX2UxOUN0Rm5hbTFoOU1Gamh5azRSMTUzNUNSd05VSWI0In0="
-    style="width: 100%; height: 100%; border: none;">
+  src="http://app.svix.com/login#key=eyJhcHBJZCI6ICJhcHBfMXRSdFlMN3pwWWR2NHFuWTRRZFI1azE4eXQ0IiwgInRva2VuIjogImFwcHNrX2UxOUN0Rm5hbTFoOU1Gamh5azRSMTUzNUNSd05VSWI0In0="
+  style="width: 100%; height: 100%; border: none; background: linear-gradient(to right, #f6f7f8 0%, #edeef1 20%, #f6f7f8 40%, #f6f7f8 100%)"
```

Comment:
Added a static "skeleton" for the iframe while it loads. Also set loading to `lazy`.

Reply:
Is there a way to make an example of how to add generic loading indicators? E.g. just a div, so that people can put whatever they want there?

Reply:
AFAIK, there are 2 options: 
- show the loading "state" behind the iframe (essentially what this example is doing with the background gradient)
- do something on the iframe's `onload` callback.

Option 2 is a bit tough to have a generic example of since it means inlining javascript that will likely look nothing like the user's codebase. It would probably be easier to just have framework-specific examples

Reply:
Can't it be achieved with something like (made up class names):
```
<div class="relative">
  <div class="absolute w-100 h-100 l-0 t-0">a</div>
  <iframe class="absolute w-100 h-100 l-0 t-0 z-index-100"></iframe>
</div>
```
?

Maybe we can have a more complex example below that just shows how to do this like this? I definitely agree that having a JS example is non-intuitive.